ui/src/alerting/actions/checks.ts

```diff
@@ -85,6 +88,7 @@ export const getChecks = () => async (
     }
 
     dispatch(setAllChecks(RemoteDataState.Done, resp.data.checks))
+    dispatch(checkChecksLimits())
```

Comment:
I wish that the API would return rate limiting information along with resource responses... feels really off that we are doing all this on the client.